Makefile.manual

```diff
@@ -4,6 +4,12 @@ FC ?= gfortran
 FFLAGS ?= -Wall -Wextra -Wimplicit-interface -fPIC -g -fcheck=all
 FYPPFLAGS ?=
 
+VERSION := $(subst ., ,$(file <VERSION))
```

Comment:
```suggestion
VERSION := $(subst ., ,$(file < VERSION))
```
This is an input redirection right?

Reply:
There is no space used in the make docs: https://www.gnu.org/software/make/manual/make.html#File-Function

Reply:
There seems to be a space here in the syntax definition `$(file op filename[,text])`, where op is one of >, <, <<. This is a nit-pick.